Build failure: obs-studio-plugins.obs-multi-rtmp

### Steps To Reproduce

Steps to reproduce the behavior:
1. build `obs-studio-plugins.obs-multi-rtmp`

### Build log

```
obs-multi-rtmp> Running phase: qtPreHook
obs-multi-rtmp> Running phase: unpackPhase
obs-multi-rtmp> unpacking source archive /nix/store/xpv8n36v3gky7h988i6vdjy7brrqq8vv-source
obs-multi-rtmp> source root is source
obs-multi-rtmp> Running phase: patchPhase
obs-multi-rtmp> Running phase: updateAutotoolsGnuConfigScriptsPhase
obs-multi-rtmp> Running phase: configurePhase
obs-multi-rtmp> fixing cmake files...
obs-multi-rtmp> cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/share/doc/obs-multi-rtmp -DCMAKE_INSTALL_INFODIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/vh9fsdhgxcnab2qk7vdp2palkkn6j3cp-gcc-wrapper-13.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/vh9fsdhgxcnab2qk7vdp2palkkn6j3cp-gcc-wrapper-13.3.0/bin/ranlib -DCMAKE_AR=/nix/store/vh9fsdhgxcnab2qk7vdp2palkkn6j3cp-gcc-wrapper-13.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/3dqxn67fgq1xmx5dgjjj3ggkw98kgrdd-obs-multi-rtmp-0.6.0.1 -DENABLE_QT:BOOL=TRUE -DENABLE_FRONTEND_API:BOOL=TRUE -DCMAKE_COMPILE_WARNING_AS_ERROR:BOOL=FALSE
obs-multi-rtmp> -- The C compiler identification is GNU 13.3.0
obs-multi-rtmp> -- The CXX compiler identification is GNU 13.3.0
obs-multi-rtmp> -- Detecting C compiler ABI info
obs-multi-rtmp> -- Detecting C compiler ABI info - done
obs-multi-rtmp> -- Check for working C compiler: /nix/store/vh9fsdhgxcnab2qk7vdp2palkkn6j3cp-gcc-wrapper-13.3.0/bin/gcc - skipped
obs-multi-rtmp> -- Detecting C compile features
obs-multi-rtmp> -- Detecting C compile features - done
obs-multi-rtmp> -- Detecting CXX compiler ABI info
obs-multi-rtmp> -- Detecting CXX compiler ABI info - done
obs-multi-rtmp> -- Check for working CXX compiler: /nix/store/vh9fsdhgxcnab2qk7vdp2palkkn6j3cp-gcc-wrapper-13.3.0/bin/g++ - skipped
obs-multi-rtmp> -- Detecting CXX compile features
obs-multi-rtmp> -- Detecting CXX compile features - done
obs-multi-rtmp> -- Found Threads: TRUE
obs-multi-rtmp> -- Performing Test HAVE_STDATOMIC
obs-multi-rtmp> -- Performing Test HAVE_STDATOMIC - Success
obs-multi-rtmp> -- Qt version found: 6
obs-multi-rtmp> -- Found OpenGL: /nix/store/vpys8vaqmjlzcn12wfccym7kgl7b3fr1-libglvnd-1.7.0/lib/libOpenGL.so
obs-multi-rtmp> -- Found WrapOpenGL: TRUE
obs-multi-rtmp> -- Found XKB: /nix/store/0j2mnm88hf6ndp58jybmbiwl4wg492g9-libxkbcommon-1.7.0/lib/libxkbcommon.so (Required is at least version "0.5.0")
obs-multi-rtmp> -- Found WrapVulkanHeaders: /nix/store/jrr54zy7h7w1lhh3rk5q4zcbfwz57r3c-vulkan-headers-1.3.290.0/include
obs-multi-rtmp> CMake Error at cmake/common/helpers_common.cmake:89 (set_property):
obs-multi-rtmp>   set_property can not be used on an ALIAS target.
obs-multi-rtmp> Call Stack (most recent call first):
obs-multi-rtmp>   CMakeLists.txt:25 (find_qt)
obs-multi-rtmp> 
obs-multi-rtmp> -- Configuring incomplete, errors occurred!
error: builder for '/nix/store/mwia79gsmk5j5w782r9hks1y2rvla7d9-obs-multi-rtmp-0.6.0.1.drv' failed with exit code 1;
       last 10 log lines:
       > -- Found WrapOpenGL: TRUE
       > -- Found XKB: /nix/store/0j2mnm88hf6ndp58jybmbiwl4wg492g9-libxkbcommon-1.7.0/lib/libxkbcommon.so (Required is at least version "0.5.0")
       > -- Found WrapVulkanHeaders: /nix/store/jrr54zy7h7w1lhh3rk5q4zcbfwz57r3c-vulkan-headers-1.3.290.0/include
       > CMake Error at cmake/common/helpers_common.cmake:89 (set_property):
       >   set_property can not be used on an ALIAS target.
       > Call Stack (most recent call first):
       >   CMakeLists.txt:25 (find_qt)
       >
       > 
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/mwia79gsmk5j5w782r9hks1y2rvla7d9-obs-multi-rtmp-0.6.0.1.drv'.
```

### Additional context

Add any other context about the problem here.

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@06kellyjac 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.111, NixOS, 24.05 (Uakari), 24.05.5366.fbca5e745367`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.8`
 - channels(root): `"nixos-24.05, nixos-hardware, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/video/obs-studio/plugins/obs-multi-rtmp/default.nix
+++ b/pkgs/applications/video/obs-studio/plugins/obs-multi-rtmp/default.nix
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, obs-studio, cmake, qtbase }:
+{ lib, stdenv, fetchFromGitHub, obs-studio, cmake, qtbase, fetchpatch }:
 
 stdenv.mkDerivation rec {
   pname = "obs-multi-rtmp";
@@ -10,6 +10,15 @@ stdenv.mkDerivation rec {
     rev = version;
     sha256 = "sha256-MRBQY9m6rj8HVdn58mK/Vh07FSm0EglRUaP20P3FFO4=";
   };
+
+  patches = [
+    # Fix finding QT. Remove after next release.
+    (fetchpatch {
+      url = "https://github.com/sorayuki/obs-multi-rtmp/commit/a1289fdef404b08a7acbbf0d6d0f93da4c9fc087.patch";
+      hash = "sha256-PDkR315y0iem1+LAqGmiqBFUiMBeEgnFW/xd1W2bAu4=";
+      includes = [ "CMakeLists.txt" ];
+    })
+  ];
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ obs-studio qtbase ];

